checkov/secrets/git_history_store.py

```diff
@@ -124,7 +124,8 @@ def get_added_and_removed_commit_hash(self, key: str, secret: PotentialSecret) -
             enriched_secrets = self.secrets_by_file_value_type[secret_key]
             chosen_secret = enriched_secrets[0]
             if len(enriched_secrets) > 1:
-                added, removed, _file = key.split("_")
+                res = key.split("_")
```

Comment:
did you change the key ?

Reply:
No. in case of files like: `my_name_have_underscore.py`
The previous code would fail, as there are 6 fields to extract and not 3.